docs/literate/src/files/differentiable_programming_literate.jl

```diff
@@ -195,14 +195,12 @@ function energy_at_final_time(k) # k is the wave number of the initial condition
     ode = semidiscretize(semi, (0.0, 1.0))
     sol = solve(ode, BS3(), save_everystep=false)
     Trixi.integrate(energy_total, sol.u[end], semi)
-end
+end;
```

Comment:
I think it's fine to have the standard output ("generic function...") in the docs since people will recognize this from the REPL. In the other hand, I would wonder why there is a `;` after the `end` of the function definition.
```suggestion
end
```
Or was there another reason for this change?

Reply:
Yeah, I think you're right. Especially because it's just one extra line.